velox/docs/functions/presto/conversion.rst

```diff
@@ -428,6 +428,8 @@ Valid examples
 
   SELECT cast('1.' as real); -- 1.0
   SELECT cast('1' as real); -- 1.0
+  SELECT cast('0x1.2p3' as real); -- 9.0
```

Comment:
Would you add some words to describe this functionality and perhaps add a link where this format is explained in detail. I think many folks may not understand what this means.